src/Worker.Extensions.DurableTask/Execution/FunctionsOrchestrator.cs

```diff
@@ -12,20 +12,20 @@ namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask;
 /// <summary>
 /// A custom orchestrator for running functions-triggered orchestrations.
 /// </summary>
-internal class FunctionsOrchestrator : ITaskOrchestrator
+internal class FunctionsOrchestrator : IDisposableOrchestrator
```

Comment:
What's the relationship between `FunctionsOrchestrator` and `WrapperOrchestrator`? It seems they have some commonalities.

Reply:
You are right there is a lot of overlap. They could possibly be combined with some effort. The main difference is what they invoke:

1. `FunctionsOrchestrator` -> existing one, invokes the function-based trigger. Uses the inner `IFunctionExecutor` to execute the function.
2. `WrapperOrchestrator` -> invokes the `ITaskOrchestrator` we got from the `IDurableTaskFactory`.

No personal opinion on combining them or not.